src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -1319,7 +1319,8 @@ private BoundExpression SynthesizeMethodGroupReceiver(CSharpSyntaxNode syntax, A
             var declaringType = members[0].ContainingType;
 
             HashSet<DiagnosticInfo> unused = null;
-            if (currentType.IsEqualToOrDerivedFrom(declaringType, TypeCompareKind.ConsiderEverything, useSiteDiagnostics: ref unused))
+            if (currentType.IsEqualToOrDerivedFrom(declaringType, TypeCompareKind.ConsiderEverything, useSiteDiagnostics: ref unused) ||
+                (currentType.IsInterface && (declaringType.IsObjectType() || currentType.AllInterfacesNoUseSiteDiagnostics.Contains(declaringType))))
```

Comment:
>( [](start = 16, length = 1)

This is not correct for an invocation of an interface method from a class nested within the interface. Although you have not implemented classes declared within interfaces yet, I do not think it makes sense to add code here that will just have to be changed again.